packages/mobile/package.json

```diff
@@ -17,10 +17,10 @@
     "dev:clean-android": "cd android && ./gradlew clean",
     "dev:send-text": "adb shell input text",
     "dev:reload": "adb shell input text \"RR\"",
-    "dev:emulator": "bash ./scripts/start_emulator.sh &",
-    "dev:remote": "remotedev --hostname=localhost --port=8000",
+    "dev:emulator:list": "$ANDROID_HOME/emulator/emulator -list-avds",
+    "dev:emulator:run": "$ANDROID_HOME/emulator/emulator -avd",
```

Comment:
I think if the functionality doesn't change it's nice to leave the name the same

Reply:
It was broken before so doubt anyone was using it. I prefer it like this